### PR TITLE
Configure TsvJob to work with latest parquet version

### PR DESCRIPTION
**Pull request by DanielleSucher**
r? @jvns or @colinmarc 

Looks like the last parquet version bump broke `herringbone tsv`. This should fix it. Manually tested it with @krivoire already, and it totally worked!

### CODE REVIEW

File: herringbone-main/src/main/scala/com/stripe/herringbone/CompactInputFormat.scala [hunk 1]
```diff
@@ -22,9 +22,8 @@ import parquet.example.data.simple.SimpleGroup
 
 class CompactInputFormat[T](readSupportClass: Class[_ <: ReadSupport[T]]) extends ParquetInputFormat[T](readSupportClass) {
 
-  // We can't accurately predict the size of the resulting merged file, so aim
-  // for 900MB. Our HDFS block size is 1024MB so we'll get pretty close.
-  val TARGET = 1024 * 1024 * 900 // 900MB.
+  // Our HDFS block size is 1024MB so we'll get pretty close.
+  val TARGET = 1024 * 1024 * 1024 // 1024MB.
```
Comment (jvns) [inline herringbone-main/src/main/scala/com/stripe/herringbone/CompactInputFormat.scala]: why this change? (line 29 still refers to the 900 MB number)

**Comment by jvns** [comment]
:+1:

File: herringbone-main/src/main/scala/com/stripe/herringbone/CompactInputFormat.scala (hunk 1 above)
Comment (DanielleSucher) [inline herringbone-main/src/main/scala/com/stripe/herringbone/CompactInputFormat.scala]: To match another internal tool that we're going to merge into this one, honestly. This fix is a move towards consistency as well. ^^ I'll fix the comment below!
